app/src/test/java/http_server/app/MessageUtilsTest.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package http_server.app;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class MessageUtilsTest {
+    @Test void testGetMessage() {
+        assertEquals("Hello      World!", MessageUtils.getMessage());
```

Comment:
Suggestion for this project. I'd encourage you to use the Hamcrest library and use the `.assertThat()` instead of `assertEquals()` from junit. I think it's more human readable

Reply:
yes! I second the suggestion to install Hamcrest - they have a wider range of assertions & I found that very useful when writing my tests for this project